docs/HowToConnectExternally.md

```diff
@@ -0,0 +1,298 @@
+
+> **NOTE**: When external access is enabled, the NuoDB Helm Charts will create Internet-facing load balancers by default.  Be sure to understand the difference between _internet facing_ load balancers (allowing connectivity external to the cloud and Kubernetes cluster) and  _internal_ load balancers (allowing connectivity external to the Kubernetes cluster but within the cloud virtual network).
+
+If the SQL clients are located outside of the Kubernetes cluster where NuoDB is deployed but in the same cloud provider or virtual network, then _internal_ load balancer can be used.
+This is configured by setting the `admin.externalAccess.internalIP=true` and `database.te.externalAccess.internalIP=true` or further customized by explicitly setting custom annotations for the Kubernetes services using the `admin.externalAccess.annotations` and `database.te.externalAccess.annotations` options.
+The user-provided custom annotations will overwrite the default annotations for the services.
+
+> **IMPORTANT**: The customer is responsible for correctly configuring security rules and restricting access to the cloud load balancers.
+
+Services of type `NodePort` can be created as well by setting the `admin.externalAccess.type=NodePort` and `database.te.externalAccess.type=NodePort` options.
+For this type of deployment, a Layer4 load balancer must be manually provisioned. It should also be configured to load balance traffic across all worker nodes in the Kubernetes cluster.
+The different TE groups will be reachable on different node ports.
+
+The external address and port for TEs are configured using the `external-address` and `external-port` process labels.
+If supplied, they will be advertised by NuoDB Admin to the SQL clients during the second stage of the client connection protocol.
+For more information, see [Use External Address](https://doc.nuodb.com/nuodb/latest/client-development/load-balancer-policies/#_use_external_address).
+
+Obtaining and configuring the address of the L4 load balancers can be tedious and error-prone as they are provisioned asynchronously by the Kubernetes controllers.
+NuoDB can inspect the Kubernetes services and configure the TE database processes with the `external-address` and `external-port` process labels automatically if the `--enable-external-access` process option is provided.
+This simplifies the deployment and ensures the correct configuration of the TE database processes.
+
+- For services of type `LoadBalancer`, NuoDB will configure the `external-address` process label with the service ingress IP or hostname as a value.
+- For services of type `NodePort` are provisioned, the customer must configure the  `external-address` and NuoDB will configure the `external-port` process label with the service node port as a value.
+If the `--enable-external-access` process option is supplied but the `external-address` process label is no defined, then external access won’t be enabled and a warning message will be logged.
```

Comment:
Should say "process label is not defined". Also, I would say "`--enable-external-access` argument" rather than "process option", because this could be conflated with process options like `mem`.

Reply:
Good catch! I will update this across the doc.